src/file.ts

```diff
@@ -2319,6 +2319,12 @@ class File extends ServiceObject<File> {
    */
   getSignedUrl(cfg: GetSignedUrlConfig, callback?: GetSignedUrlCallback):
       void|Promise<GetSignedUrlResponse> {
+    const validAction = ['read', 'write', 'delete', 'resumable'];
```

Comment:
This looks reasonable to me, might be worth considering using an `Enum` for `TypeScript`, does this translate cleanly to the JavaScript API?

Reply:
There's actually already this enum: https://github.com/googleapis/nodejs-storage/blob/7b1ef2c376e353758879050107426a980c502774/src/file.ts#L2341

We could just validate that `method` is something.

Reply:
Thanks, done